subset/connection/mac_oui/.gitignore

```diff
@@ -1 +1,3 @@
 report/
+
+/src/main/resources/macList.txt
```

Comment:
is that leading / correct?  seems like it's an absolute path to ignore, but not sure how that would work?

Reply:
I agree this would not work, it should be 

`src/main/resources/macList.txt`

@frgitdaq can you please amend it? thanks

Reply:
@grafnu @pisuke I had it without the leading / but saw that the mkdir cmd in the bacnet Dockerfile had it with leading /.

Reply:
@pisuke It did work and I double checked by creating another test file under the resources folder which did get ignored. But nonetheless I have removed the leading /